test/e2e/accordion.test.js

```diff
@@ -39,12 +39,16 @@ describe('vl-accordion', async () => {
         const openButton = await vlAccordionPage.getJSAccordionOpenButton();
         const closeButton = await vlAccordionPage.getJSAccordionCloseButton();
         const toggleButton = await vlAccordionPage.getJSAccordionToggleButton();
+        await vlAccordionPage.scrollIntoView(openButton);
```

Comment:
Moeten we dit dan niet best in de click inbouwen?

Reply:
Dit gaat direct naar de functie gedefinieerd op de demo pagina

Reply:
En ipv. page.getButton, er page.clickButton van maken en de scroll daar in steken?
Want nu is het iets dat _altijd_ moet gebeuren bij het clicken. Dan moet dat niet in de testen steken.

Reply:
Alternatief kan zijn de click overschrijven (wrap) op VlElement niveau.